src/App/Article/index.js

```diff
@@ -107,8 +108,8 @@ function Related({ id }) {
   );
 }
 
-function CitedBy({ citedBy }) {
-  return citedBy.map(id => {
+function CitedBy({ citedBy, offset }) {
+  return citedBy.slice(10 * offset, 10 * (offset + 1)).map(id => {
```

Comment:
Is the problem that the offset comes in here, and changes the number of times the hook is called, further down?